pkg/kv/kvserver/store_rebalancer.go

```diff
@@ -101,6 +102,22 @@ var qpsRebalanceThreshold = func() *settings.FloatSetting {
 	return s
 }()
 
+var loadBasedRebalanceInterval = settings.RegisterPublicDurationSettingWithExplicitUnit(
+	settings.SystemOnly,
+	"kv.allocator.load_based_rebalancing_interval",
+	"the rough interval at which each store will check for load-based lease / replica rebalancing opportunities",
+	defaultLoadBasedRebalancingInterval,
+	func(d time.Duration) error {
+		// Setting this interval to a very low duration is generally going to be a
+		// bad idea without any real benefit, so let's disallow that.
+		const min = 10 * time.Second
+		if d <= min {
```

Comment:
nit: consider making the allowable range inclusive of 10 seconds so that it's easier to set this setting to its minimum value.